stablehlo/dialect/Version.cpp

```diff
@@ -80,7 +80,7 @@ Version Version::fromCompatibilityRequirement(
     case CompatibilityRequirement::NONE:
       return Version::getCurrentVersion();
     case CompatibilityRequirement::WEEK_4:
-      return Version(1, 5, 0);  // v1.5.0 - Aug 1, 2024
+      return Version(1, 6, 0);  // v1.6.0 - Aug 16, 2024
```

Comment:
Above this switch statement, can you add a comment:

```
// The time frames used are from the date that the release was tagged on, not merged.
// The tag date is when the version has been verified and exported to XLA.
// See: https://github.com/openxla/stablehlo/tags
```

Reply:
Also bump to 1.7.0 as mentioned, that will allow new fp types to be used from JAX export, that tag date is now 4w ago.